python/networkx.cpp

```diff
@@ -686,9 +686,24 @@ void ukv::wrap_networkx(py::module& m) {
         py::arg("key"));
     g.def(
         "get_edge_data",
-        [](py_graph_t& g, ukv_key_t v1, ukv_key_t v2) { throw_not_implemented(); },
+        [](py_graph_t& g, ukv_key_t v1, ukv_key_t v2, py::object default_value) {
+            std::string default_value_str;
+            to_string(default_value.ptr(), default_value_str);
+            auto edges = g.ref().edges_between(v1, v2).throw_or_release();
+            if (!edges.size())
+                return py::reinterpret_steal<py::object>(Py_None);
```

Comment:
Why would you "steal" `Py_None`? That looks dangerous. A simple Google Search suggested at [least two strictly better variants from their documentation](https://github.com/pybind/pybind11/issues/1483).

```python
return py::none();
return py::cast<py::none>(Py_None);
```

Reply:
So they do work. And it's not the only inclusion in that file. Please check the remaining lines.

I can see why you are using `py::reinterpret_steal` to cast to `py::object`. Do you still need it if you explicitly mention the lambdas return type as `py::object`?

Reply:
Yes that's the solution!